packages/payment-detection/src/erc20/escrow-info-retriever.ts

```diff
@@ -1,6 +1,8 @@
 import { PaymentTypes } from '@requestnetwork/types';
+
```

Comment:
i wonder if this file is needed? isn't thegraph retriever enough?

Reply:
It's not needed, but it was written first, before indexing the escrow via thegraph. 
Maybe we can leave it as a failsafe just in case ?